src/client/Microsoft.Identity.Client.Desktop/Microsoft.Identity.Client.Desktop.csproj

```diff
@@ -13,7 +12,7 @@
   <PropertyGroup Label="NuGet and AssemblyInfo metadata">
     <!--This should be passed from the VSTS build-->
     <MsalClientSemVer Condition="'$(MsalClientSemVer)' == ''">4.7.1</MsalClientSemVer>
-    <MsalClientSemVer>$(MsalClientSemVer)-preview</MsalClientSemVer>
```

Comment:
We already require ".WithExperimentalFeatures" and preview packages are hard to discover